src/en/space-station-14/round-flow/proposals/time-agent.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+This proposal suggests the implementation of a new, mostly-neutral solo antagonist role, Time Agent, in order to add more variety and fun to possible encounterable gameplay. The role's theme revolves around undercover agents sent back in time or from the past to nudge the timeline in the direction that suits their goals best, be it by preventing certain events or by ensuring that they happen in the first place.
+
+
+## Mechanics
+#### Chronodisc:
+Every agent starts with a **Chronodisc** item in their bag. All abilities require the **Chronodisc** to be present somewhere in the inventory or body of their owner. Every agent is attuned to their own **Chronodisc**, so one cannot use the **Chronodisc** of another for the purposes of time manipulation. (therefore, Security might be willing to place a repentant agent on parole by taking away and destroying their **Chronodisc**, effectively making them a normal crewmember; the Science department might also appreciate being given a **Chronodisc**, as it grants a nice amount of research points)
+
+#### Possibilities:
+By completing objectives - even optional ones - agents create opportunities for them to be able to use their abilities safely. Each agent starts with 1 possibility plus 1 additional possibility per **Nudge Life** objective at spawntime. The amount of possibilities the agent may have can freely dip into the negatives, but there are consequences to this listed below.
+
+#### Doom:
+By failing mandatory objectives or by excessively using possibilities, the total amount of possibilities an agent currently has decreases. This amount can decrease into the negative. When an agent's amount of available possibilities is negative, they become doomed. Dying while doomed prevents revival and resuscitation. While an agent is alive for a certain amount of time while doomed, they become vulnerable to Doom Events. The only way for an agent to escape doom is to bring the amount of possibilities they have back to the positive or, at the very least, 0.
+
+#### Doom Events:
+When a doomed agent is alive for a certain amount of time - the timer for which gets halved every time the agent continues expending possibilities while their counter is in the negative - things around them may malfunction in certain ways to cause their demise. Examples include getting shocked having a 90% chance of husking, getting hit by a stabbing/blunt attack having a 90% chance of piercing/bludgeoning the eyes & brain if the attack was on the head, getting hit by a stabbing/blunt attack having a 90% chance of piercing/bludgeoning the heart if the attack was on anything other than the head, damage taken from chemicals/diseases/ailments is increased tenfold, being affected by a chemical substance has a 90% chance to instantly cause an allergy to said substance, etc.

```

Comment:
most of these mechanics just do not exist in the game as is, what would it actually look like

Reply:
...Perhaps I was overestimating SS14's current level of complexity.

I guess Doom Events can be simplified to a 10x multiplier to all incoming damage (both from combat and from DoT stuff like harmful chemicals) as well as a 25% chance (rolled every minute with random delays) to cause the agent to instantly die with one of several unique death messages. (and various unique flairs accompanying the message in question, like some blood decals, vomit decals, outright gibbing, loss of limbs, etc.)